keyboards/vertex/arc60h/keyboard.json

```
@@ -49,6 +49,12 @@
     "diode_direction": "COL2ROW",
     "processor": "STM32F103",
     "bootloader": "stm32duino",
+    "community_layouts": [
+        "60_ansi_tsangan",
+        "60_ansi_wkl",
+        "60_ansi_wkl_split_bs_rshift",
+        "60_hhkb"
+    ],
```

@fauxpark: not sure what future plan is for renaming CL `60_tsangan_hhkb` to `60_ansi_tsangan_split_bs_rshift`
```suggestion
    "layout_aliases": {
        "LAYOUT_60_tsangan_hhkb": "60_ansi_tsangan_split_bs_rshift"
    },
    "community_layouts": ["60_ansi_tsangan", "60_tsangan_hhkb", "60_ansi_wkl", "60_ansi_wkl_split_bs_rshift", "60_hhkb"],
```

---

I wondered why that was missing... yeah, that should probably be renamed.